### PR TITLE
Add badges to Readme file

### PR DESCRIPTION
**Pull request by azzmiks**
Problem: There are not badges in the Readme file with some statistics about the project.
Solution: Add badges to Readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
 
 This library provides a programmatic way to manage Datadog synthetic browser tests. This library contains all the necessary functions that are required to manage the synthetic browser tests with different steps and configurations.
 
+[![License](https://img.shields.io/badge/License-MIT-brightgreen)](https://github.com/personio/datadog-synthetic-test-support/blob/master/LICENSE)
+[![Version](https://img.shields.io/github/v/release/personio/datadog-synthetic-test-support)](https://github.com/personio/datadog-synthetic-test-support/releases)
+[![Release date](https://img.shields.io/github/release-date/DataDog/datadog-api-client-java)](https://github.com/personio/datadog-synthetic-test-support/releases)
+![Downloads](https://img.shields.io/github/downloads/personio/datadog-synthetic-test-support/total)
```
Comment (emanuelsantan4) [inline README.md:8]: The download does not have a URL for it. Is that correct? Will there be an URL at some point?

**Review by emanuelsantan4 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -2,6 +2,25 @@
...
+
+[![Verify Build](https://img.shields.io/github/actions/workflow/status/personio/datadog-synthetic-test-support/verify.yml?label=verify%20build)](https://github.com/personio/datadog-synthetic-test-support/actions/workflows/verify.yml)
+[![Release Build](https://img.shields.io/github/actions/workflow/status/personio/datadog-synthetic-test-support/release-and-publish.yml?label=release%20build)](https://github.com/personio/datadog-synthetic-test-support/actions/workflows/release-and-publish.yml)
+[![Open issues](https://img.shields.io/github/issues-raw/personio/datadog-synthetic-test-support)](https://github.com/personio/datadog-synthetic-test-support/issues?q=is%3Aopen+is%3Aissue)
+[![Closed issues](https://img.shields.io/github/issues-closed-raw/personio/datadog-synthetic-test-support)](https://github.com/personio/datadog-synthetic-test-support/issues?q=is%3Aissue+is%3Aclosed)
+
+![Top Language](https://img.shields.io/github/languages/top/personio/datadog-synthetic-test-support)
```
Comment (arunprakashind) [inline README.md:15]: I think we can avoid these badges or probably push them to the bottom of the readme. It applies to the next section as well

**Review by arunprakashind — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -2,6 +2,25 @@
 
 This library provides a programmatic way to manage Datadog synthetic browser tests. This library contains all the necessary functions that are required to manage the synthetic browser tests with different steps and configurations.
 
+[![License](https://img.shields.io/badge/License-MIT-brightgreen)](https://github.com/personio/datadog-synthetic-test-support/blob/master/LICENSE)
+[![Version](https://img.shields.io/github/v/release/personio/datadog-synthetic-test-support)](https://github.com/personio/datadog-synthetic-test-support/releases)
+[![Release date](https://img.shields.io/github/release-date/DataDog/datadog-api-client-java)](https://github.com/personio/datadog-synthetic-test-support/releases)
```
[the remark this answers is not in the document]
Comment (azzmiks) [reply README.md]: Was using it to create badges while the repo was private. Fixed

File: README.md (hunk 2 above)
Comment (azzmiks) [reply README.md]: Fixed

**Review by arunprakashind — APPROVED** [review]

**Pull request merged**